src/docs/background/privileged-transitions/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Privileged transitions
+slug: privileged-transitions
+updated: 2020-06-16
+category: background
+ingress:
+  This article introduces you to the concept of privileged transitions
+  and how Flex uses them to invoke actions from a secure context.
+published: true
+---
+
+_**NOTE! Privileged transitions enable highly flexible pricing schemes,
+deprecating our custom pricing feature that was in the past used for
+price customizations. If you are looking for the old custom pricing
+article, it can be found
+[here](https://5ee94c280d38f10008a3bfa1--sharetribe-flex-docs-site.netlify.app/docs/background/custom-pricing/).**_
+
+## What are privileged transitions?
+
+In Flex, a process transition is an edge between two states in the
+transaction process graph. Invoking transitions is guarded in the
+process definition by tying them to a specific state when they can be
+transitioned and by defining which party of a transaction can perform
```

Comment:
"which party" is not technically correct because we have 4 different actor roles but in this context I think it's still right enough and keeps things simpler so this is just a note but not a change request.

Reply:
Indeed, my thinking was that this implies that the transitioning actor can only be provider or customer as it makes sense in this context even though there are more actor roles. I'll see if this could be changed in a way that does not make it more complex but does not confuse users who are aware of the four different actors.